search-commons/src/main/java/no/unit/nva/search/common/records/Midas.java

```diff
@@ -0,0 +1,7 @@
+package no.unit.nva.search.common.records;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface Midas {
+    JsonNode transform(JsonNode source);
```

Comment:
Whats a Midas? I havent heard about it before. Tried googling and still dont understand

Reply:
Midas transformed everything into gold...